doc/whats-new.rst

```diff
@@ -25,15 +25,30 @@ Breaking changes
 - xarray no longer supports python 3.6
 
   The minimum version policy was changed to also apply to projects with irregular
-  releases.
-
-  The minimum versions of some other dependencies were changed:
+  releases. As a result, the minimum versions of some dependencies were changed:
 
   ============ ====== ====
   Package      Old    New
   ============ ====== ====
   Python       3.6    3.7
   setuptools   38.4   40.4
+  numpy        1.15   1.17
+  pandas       0.25   1.0
+  dask         2.9    2.11
+  distributed  2.9    2.11
+  boto3        1.9    1.12
+  bottleneck   1.2    1.3
+  h5netcdf     0.7    0.8
+  h5py         0.9    2.10
+  iris         2.2    2.4
+  lxml         4.4    4.5
+  netcdf4      1.4    1.5
+  numba        0.46   0.48
+  pseudonetcdf 3.0    3.1
+  rasterio     1.0    1.1
+  scipy        1.3    1.4
+  seaborn      0.9    0.10
+  zarr         2.3    2.4
```

Comment:
how do we feel about documenting all bumped dependencies? There are also a few which are indirect dependencies (`lxml`, `numba`, and `boto3` (?))

Reply:
No strong view — I think it's fine to include them if it's easy, but definitely no need to type them out. Particularly on the indirect ones.

(Is boto3 a required dependency? :) )

Reply:
I removed the indirect dependencies so the table should be a bit smaller.

> Is boto3 a required dependency

indirect, I think? It is only mentioned in the environment files.